README.md

```diff
@@ -2,18 +2,56 @@ its-client
 ==========
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![Build Status](https://github.com/Orange-OpenSource/its-client/workflows/Docker/badge.svg)](https://github.com/Orange-OpenSource/its-client/actions/workflows/docker.yml)
 
-This Intelligent Transportation Systems (ITS) [MQTT][1] client based on
-the [JSON][2] [ETSI][3] specification transcription provides a ready to connect project for the mobility
-(connected and autonomous vehicles, road side units, vulnerable road users,...).
+This repository provides reference implementation of message exchanging for IoT3 over [MQTT][1] (v5)  
+On top of this it also provides a proposal of implementation for V2X messages in [JSON][2] based on [ETSI][3]'s
+Intelligent Transportation Systems (ITS)
 
-Let's connect your device or application to our Intelligent Transport Systems (ITS) platform!
+For both of these use cases telemetry is both automatically done and abstracted to send custom traces  
```

Comment:
Spurious double trailing space.

Reply:
That's one markdown way to line break, but it appears it should also be possible using a backslash, I'll try it (I agree trailing spaces are not the best way to line break)

Reply:
Oh and there is nothing to line break here in fact...

Reply:
Yes, that was the issue: there is no reason to force a line break here as this is already the end of a paragraph.

Reply:
Also, to be clear: using trailing spaces to force a line break was perfectly acceptable (I see you also changed the other s to use a backslash `\`, which is not what this comment was about, sorry for the confusion...)